hw_classes/classes_hw.py

```diff
@@ -0,0 +1,265 @@
+# Task 1 -----------------------------------------------------------------
+
+class Chat(object):
+    '''
+    Class for the chat object.
+    '''
+    def __init__(self, name, messages):
+        # stores messages from new to old (inverted order) 
+        self.chat_name = name
+        self.chat_history = messages[::-1]
+    
+    def show_last_message(self):
+        # prints the last message
+        self.chat_history[0].show()
+    
+    def get_history_from_the_timeperiod(self, start_date, end_date):
```

Comment:
Аргументы должны быть опциональными, то есть метод должен работать без одного из них или без обоих сразу

![image](https://user-images.githubusercontent.com/48160953/225777792-362d2f83-c38a-4388-8a16-5d323e18c3d8.png)

Можно поставить по-умолчанию `None`, а внутри функции уже эти случаи как-то обрабатывать
```suggestion
    def get_history_from_the_timeperiod(self, start_date=None, end_date=None):
```